docs/getting-started/dev-docker-setup.md

```diff
@@ -57,24 +60,25 @@ has_toc: true
   invenio-cli services setup
   ```
   IMPORTANT: if services setup reported any errors, and you have to restart the setup process, make sure to run
-  destroy service command first (or you will get "Failed to setup services" error ) and delete db files by running `invenio-cli services destroy` and `rm -r app_data/db/*`
+  destroy service command first (or you will get `Failed to setup services` error ) and delete db files by running `invenio-cli services destroy` and `rm -r app_data/db/*`
 
 9. Add local admin user
   ```sh
   invenio-cli shell
   invenio users create admin@test.com --password=123456 --active
   invenio roles add admin@test.com admin
   ```
-10. If you do any local UI customization you need to rebuild applications web assets
+10. Start the application.
   ```sh
-  invenio-cli assets build --development
+  invenio-cli run
   ```
-
-11. Start the application.
+11. If you do any local UI customization you need to rebuild applications web assets
```

Comment:
it might be helpful here to indicate which step one would need to go back to to rebuild the assets. It's `invenio-cli services setup` right?

Reply:
nevermind.. i see it below